repository/gogit.go

```diff
@@ -182,7 +182,7 @@ func (repo *GoGitRepo) GlobalConfig() Config {
 	// TODO: replace that with go-git native implementation once it's supported
 	// see: https://github.com/go-git/go-git
 	// see: https://github.com/src-d/go-git/issues/760
-	return newGoGitGlobalConfig(repo.r)
+	return newGitConfig(gitCli{path: repo.path}, true)
```

Comment:
I wanted to share my bridge configuration against multiple projects. Also, there is a bug in the marshalling which is deleting the anti-slashes from the values.

Reply:
> There is a bug in the marshalling which is deleting the anti-slashes from the values. I've replaced with the git cli implementation. This should be solved in the go library. Didn't reported yet.

Please report it, it could cause many problems.